Vic2ToHoI4Tests/HoI4WorldTests/Leader/CountryLeadersFactoryTests.cpp

```diff
@@ -0,0 +1,13 @@
+#include "HOI4World/Leaders/CountryLeadersFactory.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+
+
+TEST(HoI4World_Leaders_CountryLeadersFactoryTests, LeadersCanBeImported)
+{
+	const auto leaders = HoI4::CountryLeadersFactory{}.importCountryLeaders();
+
+	ASSERT_EQ(1, leaders.count("ONE"));
+	ASSERT_EQ(2, leaders.count("TWO"));
```

Comment:
Which of the two TWO leaders is loaded?

Reply:
Both are. I can't guarantee the order in the container, and don't want to bother creating an equality operator to enable the fancy container comparisons, thus only checking the number.